R/formatter.r

```diff
@@ -74,32 +72,25 @@ number <- function(x, accuracy = 1, scale = 1, prefix = "",
   ret
 }
 
-#' Comma formatter: format number with commas separating thousands.
-#'
-#' @param ... other arguments passed on to [format()]
-#' @param x a numeric vector to format
-#' @return a function with single parameter x, a numeric vector, that
-#'   returns a character vector
+#' `comma_format` and `comma` format numbers with commas separating thousands.
```

Comment:
I think it's easier to understand if you put all the description in one place. 

(But the other tags should stay with individual functions)

Reply:
ok. I have also reordered the code (i.e. each function being declared in the R file in the same order as the doc).